arviz/data/inference_data.py

```diff
@@ -326,18 +328,35 @@ def from_netcdf(filename: str) -> "InferenceData":
         ----------
         filename : str
             location of netcdf file
+        group_kwargs : dict of dict
+            Keyword arguments to be passed into each call of `xarray.open_dataset`.
```

Comment:
```suggestion
        group_kwargs : dict of {str: dict}, optional
            Keyword arguments to be passed into each call of {func}`xarray.open_dataset`.
```

the first change is a nit to match numpydoc docstring preferred style, the second will make the xarray.open_dataset be a link to xarray docs.